src/components/floatingButton/__tests__/index.spec.tsx

```diff
@@ -19,7 +19,7 @@ const TestCase = (props) => {
 };
 export const FloatingButtonDriver = (props: ComponentProps) => {
   const driver = useComponentDriver(props);
-  const getStyle = () => driver.getProps().style as ViewStyle;
+  const getStyle = () => driver.getElement().props.style as ViewStyle;
```

Comment:
Isn't props optional? Don't you need to do `driver.getElement().props**?**.style`?

Reply:
I think they're not. Its always a component that is captured so it has testId so props exists.

Reply:
Plus according to the type `ReactTestInstance` props is not optional so they probably promise us there will be props.

Reply:
Ok...